azerothcore/azerothcore-wotlk
# fix(DB/spell_custom_attr):  Debuffs incorrectly applied as buffs

## CHANGES PROPOSED:
-  Change 'Phase Punch' to correctly be applied as a debuff on Algalon 10 and 25 in Ulduar
-  Change 'Death Coil' to correctly be applied as a debuff on High King Maulgar in Gruul's Lair

## ISSUES ADDRESSED:
- #3149 placed spells to be modified as positive/negative within the database; but the original PR set spells that are supposed to be negative as positive. 


## TESTS PERFORMED:
- Deleting the row in the table set 'Phase Punch' as a debuff
- Adding row for set 'Death Coil' as debuff


## HOW TO TEST THE CHANGES:
- Apply SQL, restart server, create character
- Run Ulduar 10 or 25 and pull Algalon, check if Phase Punch is a debuff or not
- Run Gruul's Lair and wait for Death Coil to be cast on you

## Target branch(es):
